File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ActivityconvertOptionsToString.cpp
#include "ActivityconvertOptionsToString.h"
#include <string>
#include "RadioButton_ViewModelOptionsOption.h"

namespace widgetassertions
{
  int ActivityconvertOptionsToString::convertOptionsToString(RadioButton_ViewModelOptionsOption symbol)
  {
    if (symbol == widgetassertions::RadioButton_ViewModelOptionsOption::OptionA)
    {
      return std::string("OptionA");
    }

    if (symbol == widgetassertions::RadioButton_ViewModelOptionsOption::OptionB)
    {
      return std::string("OptionB");
    }

    if (symbol == widgetassertions::RadioButton_ViewModelOptionsOption::OptionC)
    {
      return std::string("OptionC");
    }

    return std::string("?");
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ActivityconvertOptionsToString.cs
/// <filename>
///     ActivityconvertOptionsToString.cs
/// </filename>
using widgetassertions;

namespace widgetassertions
{
    public class ActivityconvertOptionsToString
    {
        public static int convertOptionsToString(widgetassertions.RadioButton_ViewModelOptionsOption symbol)
        {
            if (symbol == widgetassertions.RadioButton_ViewModelOptionsOption.OptionA)
            {
                return "OptionA";
            }

            if (symbol == widgetassertions.RadioButton_ViewModelOptionsOption.OptionB)
            {
                return "OptionB";
            }

            if (symbol == widgetassertions.RadioButton_ViewModelOptionsOption.OptionC)
            {
                return "OptionC";
            }

            return "?";
        }
    }
}
